src/server.ts

```diff
@@ -0,0 +1,65 @@
+import fastify from 'fastify';
+import path from 'path';
+import { loadActionPlugins } from './components/postgresConnect';
+import {
+  saveFiles,
+  getFilename,
+  createFilesFolder,
+  filesFolderName,
+} from './components/fileHandler';
+import getAppRootDir from './components/getAppRoot';
+
+// Bare-bones Fastify server
+
+const startServer = async () => {
+  await loadActionPlugins(); // Connects to Database and listens for Triggers
+
+  createFilesFolder();
+
+  const server = fastify();
+
+  server.register(require('fastify-static'), {
```

Comment:
Can `require('fastify-static')` be imported as one of the packages on the top please?

Reply:
Yes!